packages/next-server/server/next-server.ts

```diff
@@ -224,17 +230,22 @@ export default class Server {
     return this.sendHTML(req, res, html)
   }
 
+  private async renderHTML(req: IncomingMessage, res: ServerResponse, pathname: string, query: ParsedUrlQuery = {}, opts: any) {
```

Comment:
There're functions `this.renderHTML`, `renderToHTML` and `this.renderToHTML`, the code is very clear but I'm just a little bit confused by the names. 😅